### ISSUE TITLE
Starting a new experiment while another one is already running

### ISSUE THREAD

**Issue by mipo57** [iterative/dvc#5715]
It would be great if we could run a new experiment (with --temp) while another one is already being executed (or at least be able add a new experiment to an executed queue). This way we could use e.g. ray tune / optuna sweeper to add new runs with hyperparameters set based on already evaluated trials (eg. Bayesian Optimization).

If I understand correctly, currently the only option is to wait for the whole queue to execute before we can add new experiments (and that leaves a lot of GPU's idle :P )

**Comment by ciaochiaociao** [comment]
Really want this feature too !

**Comment by pmrowla** [comment]
Closing this in favor of https://github.com/iterative/dvc/issues/5615

See https://github.com/iterative/dvc/issues/5718#issuecomment-810203980 for further explanation
